libc-test/build.rs

```diff
@@ -3525,18 +3527,26 @@ fn config_gnu_bits(target: &str, cfg: &mut ctest::TestGenerator) {
 }
 
 fn test_linux(target: &str) {
```

Comment:
Could you add a note here about how this covers l4re as well since they have similar API surface?

Reply:
done